### PR TITLE
fix: dump generate invalid yaml

### PR DESCRIPTION
**Pull request by jtherin**
Fixes https://github.com/kubeflow/kfctl/issues/412

### CODE REVIEW

File: pkg/kfapp/kustomize/kustomize.go [hunk 1]
```diff
@@ -216,7 +216,8 @@ func (kustomize *kustomize) Dump(resources kftypesv3.ResourceEnum) error {
 		if err != nil {
 			return err
 		}
-		fmt.Println(string(data))
+		fmt.Print(string(data))
```
Comment (PatrickXYS) [inline pkg/kfapp/kustomize/kustomize.go:219]: Add `---` is okay, but I do want to keep printing original data with new line.

**Review by PatrickXYS — COMMENTED** [review]
@jtherin Thanks for contributing! 

I left a few comments

File: pkg/kfapp/kustomize/kustomize.go (hunk 1 above)
Comment (jtherin) [reply pkg/kfapp/kustomize/kustomize.go]: Okay, I've updated the PR :).

**Pull request merged**